app/views/viewlet-manager.js

```diff
@@ -316,12 +326,39 @@ YUI.add('juju-viewlet-manager', function(Y) {
             // parameter needed to be added so that multiple views could
             // be rendered into a single container
             if (!viewToCheck.slot) {
-              viewToCheck.hide();
+              // If this is a React component then it doesn't have a hide
+              if (typeof viewToCheck.hide === 'function') {
```

Comment:
yea, we need something like a Y.ReactMgr that we can check so these are more explicit. Then we'd just check that if we had a ReactMgr we'd do X vs Y